types/ensure/package.pp

```diff
@@ -0,0 +1,2 @@
+# @summary Validate the value of the ensure parameter for a package
+type Stdlib::Ensure::Package = Variant[Enum['present', 'absent', 'purged', 'disabled', 'installed', 'latest'], String[1]]
```

Comment:
This may as well be:
```suggestion
type Stdlib::Ensure::Package = String[1]
```
The Enum is completely superfluous and im not sure there is a whole lot of use in having a type definitions which is just an alias to String[1].  no objections either though

Reply:
@arjenz @b4ldr The Enum is, indeed, pretty much an alias for String[1]. However, we prefer to have it defined that way for a couple of reasons. One is to make the parameter easy to understand, by displaying valid values right away, and the other is to keep the documentation (see REFERENCE.md) as accurate as possible regarding $ensure values.

Reply:
ack no  blocker from me